src/plugins/discover/public/application/main/components/layout/use_discover_histogram.ts

```diff
@@ -279,7 +279,10 @@ export const useDiscoverHistogram = ({
         textBasedFetchComplete$.pipe(map(() => 'discover'))
       ).pipe(debounceTime(50));
     } else {
-      fetch$ = stateContainer.dataState.fetch$.pipe(map(() => 'discover'));
+      fetch$ = stateContainer.dataState.fetch$.pipe(
+        filter(({ options }) => !options.fetchMore), // don't update histogram for "Load more" in the grid
```

Comment:
Do any of the histogram unit tests cover this case?

Reply:
@weltenwort Added via https://github.com/elastic/kibana/pull/157241/commits/a57eafa030f56deafba0cd1519111ef31f53fee8, thanks!